### ISSUE TITLE
Tried all the tips to fix "Entry Error, Invalid Username or Password"

### ISSUE THREAD

**Issue by cyberskier** [gcobb321/icloud3_v3#369]
I'm sure this is due to user error on my part, but I've been trying for the last 2 hours to just get past the login screen in the setup. I enter the email address for my Apple ID, and my app-specific password (I assume I use this and not my main account password), and get "Entry Error, Invalid Username or Password"

I saw the post on Apple's Secure Remote Password verification protocol, and have tried different steps I've seen here, including:

- changing my Apple ID password, which revokes all the app-specific passwords, and getting a new password
- trying a new app-specific password with and without the dashes
- deleting and reinstalling iCloud3
- deleting iCloud3, deleting the .storage folders (iCloud, icloud3, and the icloud3 accounts one) and reinstalling
- other things I can't remember because I've been at this for a while now

Any other suggestions? There is nothing in the Event Log, other than "The following issues were detected when starting iCloud3.
Scroll through the Startup Log for more information: 1. iCloud Configuration not set up"

When I use the Gear icon to select the Startup Log, all I see different is a green box with "Start up log, alerts and èrrors are displayed → Refresh to close."

Thank you for this integration, and for any help.

**Comment by cyberskier** [comment]
So, I just kept trying and trying, and finally was able to log in. This is very slick!

**Comment by gcobb321** [comment]
Apple does not support App Specific Passwords with web based programs like iCloud3. You have to use your actual password.

**Comment by cyberskier** [comment]
Yes, I meant to come back and say that I switched to using the regular password, and went through the process of trying to get it working, with no luck for quite a while. It still gave me the error several times, and it finally just worked on my last try before bed. The only thing I did differently was try it on a different device the time it worked, but I can’t imagine how that would matter.

Anyway, this is quite amazing. I’m still just dipping my toes in the water this morning.
